eva_submission/eload_backlog.py

```diff
@@ -38,24 +44,59 @@ def get_species_info(self):
         """Adds species info into the config: taxonomy id and scientific name,
         and assembly accession, fasta, and report."""
         with get_metadata_conn() as conn:
-            query = f"select a.taxonomy_id, b.scientific_name, d.assembly_accession " \
+            query = f"select a.taxonomy_id, b.scientific_name " \
                     f"from project_taxonomy a " \
                     f"join taxonomy b on a.taxonomy_id=b.taxonomy_id " \
-                    f"join assembly_set c on b.taxonomy_id=c.taxonomy_id " \
-                    f"join accessioned_assembly d on c.assembly_set_id=d.assembly_set_id " \
                     f"where a.project_accession='{self.project_accession}';"
             rows = get_all_results_for_query(conn, query)
-        if len(rows) != 1:
+        if len(rows) < 1:
             raise ValueError(f'No taxonomy for {self.project_accession} found in metadata DB.')
-        tax_id, sci_name, asm_accession = rows[0]
+        elif len(rows) > 1:
+            raise ValueError(f'Multiple taxonomy for {self.project_accession} found in metadata DB.')
+        tax_id, sci_name = rows[0]
         self.eload_cfg.set('submission', 'taxonomy_id', value=tax_id)
         self.eload_cfg.set('submission', 'scientific_name', value=sci_name)
-        self.eload_cfg.set('submission', 'assembly_accession', value=asm_accession)
 
+        with get_metadata_conn() as conn:
+            query = f"select distinct b.vcf_reference_accession " \
+                    f"from project_analysis a " \
+                    f"join analysis b on a.analysis_accession=b.analysis_accession " \
+                    f"where a.project_accession='{self.project_accession}';"
+            rows = get_all_results_for_query(conn, query)
+        if len(rows) < 1:
+            raise ValueError(f'No reference accession for {self.project_accession} found in metadata DB.')
+        elif len(rows) > 1:
+            raise ValueError(f'Multiple reference accession for {self.project_accession} found in metadata DB.')
+        asm_accession, = rows[0]
```

Comment:
```suggestion
        asm_accession = rows[0]
```

Reply:
I'm receiving a tuple of one element so I need the comma to get the element on its own.